internal/config/config_test.go

```diff
@@ -130,3 +131,42 @@ projects:
 		})
 	}
 }
+
+func TestConfig_CachePath(t *testing.T) {
+	tests := []struct {
+		name     string
+		path     string
+		expected string
+	}{
+		{
+			name:     "with no .infracost directory",
+			path:     "testdata/no_cache_init",
+			expected: "testdata/no_cache_init",
+		},
+		{
+			name:     "with parent cache",
+			path:     "testdata/parent_cache_path/parent/child",
+			expected: "testdata/parent_cache_path/parent",
+		},
+		{
+			name:     "with root cache path",
```

Comment:
I don't understand this test case

Reply:
it's to define if the `.infracost` directory is already in the folder it shouldn't traverse the parent directories - i'll rename